src/Interpreters/InterpreterCreateQuery.cpp

```diff
@@ -1087,8 +1087,8 @@ BlockIO InterpreterCreateQuery::createTable(ASTCreateQuery & create)
     // If this is a stub ATTACH query, read the query definition from the database
     if (create.attach && !create.storage && !create.columns_list)
     {
-        auto database = DatabaseCatalog::instance().getDatabase(database_name);
-        if (database->shouldReplicateQuery(getContext(), query_ptr))
+        auto database = DatabaseCatalog::instance().tryGetDatabase(database_name);
```

Comment:
Add a comment with reasoning (you can copy-paste everything from PR description).